agent/exec/jail_errors.go

```
@@ -0,0 +1,21 @@
+// ABOUTME: Shared error sentinels for the writable_paths fs-jail (issue #272).
+// ABOUTME: Used by both Linux jail implementation and non-Linux passthrough stubs.
+package exec
+
+import "errors"
+
+// ErrLandlockUnavailable is returned by ProbeLandlock when the host kernel
+// doesn't support Landlock ABI v3 (kernel 6.7+), or when the binary is built
+// for a non-Linux target. The codergen handler refuses to start a session
+// with non-empty WritablePaths on this error.
+var ErrLandlockUnavailable = errors.New("Landlock ABI v3 not available on this host (requires Linux kernel 6.7+)")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)